### DISCUSSION TITLE
Is it possible to hide the app into windows notification area?

### DISCUSSION THREAD

**Discussion by santokalayil** [fltk-rs/fltk-rs#700] [Q&A] [answered]
Is it possible to hide the app into windows notification area? I was trying to make an app for screen recording.. I need to hide the window soon after it started recording the screen and bring back back the window soon after the recording completes.

**Comment by MoAlyousef** [comment, marked as answer]
You would have to wrap your app in a systray app like here:
https://github.com/fltk-rs/demos/tree/master/systray
